docs/02.API-REFERENCE.md

```diff
@@ -2293,6 +2381,137 @@ jerry_value_get_type (const jerry_value_t value);
 
 - [jerry_type_t](#jerry_type_t)
 
+## jerry_object_get_type
+
+**Summary**
+
+Returns the **object** type
+for a given value as a [jerry_object_type_t](#jerry_object_type_t) enum value.
+
+Note: the returned type can be checked for more detailed type information in the following cases:
+  - `JERRY_OBJECT_TYPE_CONTAINER`, *see also:* [jerry_container_type_t](#jerry_container_type_t)
+  - `JERRY_OBJECT_TYPE_FUNCTION`, *see also:* [jerry_function_type_t](#jerry_function_type_t)
+  - `JERRY_OBJECT_TYPE_ITERATOR`, *see also:* [jerry_iterator_type_t](#jerry_iterator_type_t)
+  - `JERRY_OBJECT_TYPE_TYPEDARRAY`, *see also:* [jerry_typedarray_type_t](#jerry_typedarray_type_t)
+
+**Prototype**
+
+```c
+jerry_object_type_t
+jerry_object_get_type (const jerry_value_t value);
+```
+
+- `value` - JavaScript value to check.
+- return value
+  - One of the [jerry_object_type_t](#jerry_object_type_t) value.
```

Comment:
a value from the jerry_object_type_t enumeration
there are other similar cases

Reply:
I'd leave as it to be consistent with `jerry_type_t`'s description.